x-pack/solutions/search/plugins/search_homepage/common/doc_links.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DocLinks } from '@kbn/doc-links';
+
+class ESDocLinks {
+  public kibanaFeedback: string = '';
+  public elasticCommunity: string = '';
+  public elasticsearchGettingStarted: string = '';
+  public visitSearchLabs: string = '';
+  public notebooksExamples: string = '';
+  public customerEngineerRequestForm: string = '';
+  constructor() {}
+
+  setDocLinks(newDocLinks: DocLinks) {
+    this.kibanaFeedback = newDocLinks.kibana.feedback;
+    this.elasticCommunity = newDocLinks.searchHomepage.elasticCommunity;
+    this.elasticsearchGettingStarted = newDocLinks.elasticsearch.gettingStarted;
```

Comment:
Since I think this isn't the homepage associated with the intial getting started steps, I think it's better to link to the https://www.elastic.co/docs/solutions/search landing page rather than https://www.elastic.co/docs/solutions/search/get-started
```suggestion
    this.elasticsearchDocs = newDocLinks.elasticsearch.docsBase;
```